src/utils/getBreadcrumb.js

```diff
@@ -0,0 +1,27 @@
+export const getBreadcrumb = (treeNodes, targetId) => {
+	const pathTitle = [];
+	const pathId = [];
```

Comment:
음 두 개의 배열도 좋은데, 하나의 배열 안에 title과 id를 객체 형식으로 저장하는 건 어떤가요? 아래와 같은 방식이 되겠죠?
```suggestion
    const path = [];

    const recursion = (nodes, targetId) => {
        for (const node of nodes) {
            const { id, title, documents } = node;

            path.push({ id, title });

            if (targetId === id) return true;
            if (documents.length && recursion(documents, targetId)) return true;

            path.pop();
        }
        return false;
    };
```